icyci.go

```diff
@@ -388,7 +388,9 @@ func pushResults(ch chan<- error, sourceDir string,
 			}
 		}
 
-		gitArgs := []string{"push", resultsRemote, res.ns,
+		// force push because 'branch' we fetched previously may be
+		// rebased, i.e. non-fast-forward
+		gitArgs := []string{"push", "--force", resultsRemote, res.ns,
 			stdoutNotesRef, stderrNotesRef}
 		if pushSrcToRslts {
 			gitArgs = append(gitArgs, "HEAD:refs/heads/"+branch)
```

Comment:
Thanks for the submission, @Werkov ! Instead of forcing all refs, can we get away with only forcing the `pushSrcToRslts` branch? E.g.
```
+                       // force push because 'branch' we fetched previously may
+                       // be rebased, i.e. non-fast-forward
+                       gitArgs = append(gitArgs, "+HEAD:refs/heads/"+branch)
```

Reply:
Wow, I didn't know the `+` notation. Force-pushed an update.